### ISSUE TITLE
Where are the Fortran communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/fortran#22]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Fortran supported by one or more large organizations?
* Does Fortran have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Fortran? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Fortran? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Fortran? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Fortran?
* Are there popular newsletters for Fortran?
* Is Fortran taught at programming bootcamps? (If so, what are some examples?)
* Is Fortran taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Fortran?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by kyrsjo** [comment]
You may be interested in the standards committee: https://wg5-fortran.org/

**Comment by kytrinyx** [comment]
> the standards committee

Very cool, thank you!

**Comment by anjohan** [comment]
The  usenet newsgroup (yes, this confirms another old-fashioned-fortranners-prejudice) [comp.lang.fortran](https://groups.google.com/d/forum/comp.lang.fortran) seems very active, with many experts contributing.

**Comment by jeffhammond** [comment]
# Is Fortran supported by one or more large organizations?

Intel, IBM, Cray, PGI (NVIDIA), and other vendors have a Fortran compiler product.  See http://fortranwiki.org/fortran/show/Fortran+2008+status for unofficial details.

# Does Fortran have an official community manager?

Fortran is an ISO language.  https://wg5-fortran.org/ is the organization responsible for curating the language definition.

# Do you know of specific communities (online or offline) that are enthusiastic about Fortran?

It's not possible to provide anything resembling an inclusive list.

Thousands of scientific programmers are enthusiastic about Fortran.  Fortran is heavily used by chemists, physicists, and engineers.

# Is Fortran taught at universities? (If so, what are some examples?)

* https://www.ed.ac.uk/information-services/help-consultancy/is-skills/catalogue/program-op-sys-catalogue/fortran-intro
* http://people.ds.cam.ac.uk/nmm1/Fortran/index.html
* https://www.atmos.illinois.edu/courses/atmos502-sp2017/Fortran-language.html
* https://www.tacc.utexas.edu/education/academic-courses ("Introduction to Scientific Programming")

**Comment by kytrinyx** [comment]
Thanks, this is great information!
